src/inspector_agent.cc

```diff
@@ -240,7 +240,7 @@ class V8NodeInspector : public blink::V8Inspector {
   bool running_nested_loop_;
 };
 
-Agent::Agent(Environment* env) : port_(9229),
+Agent::Agent(Environment* env) : port_(0),
```

Comment:
The port value is actually provided at the time `Agent::Start` is called. This happened to be the default initialization of the field and the value of 9229 was never getting used. This clean it up to be zero-initialized instead.